blob/gcsblob/gcsblob.go

```diff
@@ -611,6 +691,10 @@ func (b *bucket) SignedURL(ctx context.Context, key string, dopts *driver.Signed
 		PrivateKey:     b.opts.PrivateKey,
 		SignBytes:      b.opts.SignBytes,
 	}
+	// Avoid (needlessly) calling MakeSignBytes to not trigger any lazy initialization.
```

Comment:
If you do the above, then this can be simplified to `if b.ops.MakeSignBytes != nil`.